v1.0/explain.md

```diff
@@ -4,10 +4,24 @@ summary: The EXPLAIN statement provides information you can use to optimize SQL
 toc: false
 ---
 
-The `EXPLAIN` [statement](sql-statements.html) returns CockroachDB's query plan to execute [`DELETE`](delete.html), [`INSERT`](insert.html), [`SELECT`](select.html) or [`UPDATE`](update.html) statements. You can then use this information to optimize those queries.
+The `EXPLAIN` [statement](sql-statements.html) returns CockroachDB's query plan to execute the [explainable statements](#explainable-statments). You can then use this information to optimize those queries.
 
 <div id="toc"></div>
 
+## Explainable statements
+
+You can use `EXPLAIN` on the following statements:
+
+- [`ALTER TABLE`](alter-table.html)
```

Comment:
I think all the forms of ALTER can be explained, not just ALTER TABLE. Or perhaps that was not true in 1.0?

Reply:
I would also put the various ALTER statements on the same line with proper doc links like this:

```
- [`ALTER TABLE`](alter-table.html), [`ALTER INDEX`](alter-index.html), ...
```

Reply:
According to the full SQL grammar, only ALTER TABLE is explainable in 1.0. I'll update the following versions with how you suggested though.